styleguide.css

```diff
@@ -0,0 +1,273 @@
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskmedium_italic';
+    src: url('./fonts/vremenagroteskmediumitalic-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskmediumitalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+/* Sporting grotesque */
+
+@font-face {
+    font-family: 'sporting_grotesque';
+    src: url('./fonts/sporting_grotesque_gras-webfont.woff2') format('woff2'),
+    url('./fonts/sporting_grotesque_gras-webfont.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'sporting_grotesque';
+    src: url('./fonts/sporting_grotesque_normal-webfont.woff2') format('woff2'),
+    url('./fonts/sporting_grotesque_normal-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+/* General */
+
+body,
+button,
+input,
+select,
+optgroup,
+textarea {
+    color: #00283C;
+    font-family: 'vremena_groteskregular', sans-serif;
+    font-size: 19px;
+    line-height: 1.5;
+
+}
+
+/* Titles */
+
+h1 {
+    font-size:42px;
+    margin: 40px 0;
+  }
+
+h2{
+    font-size:32px;
+    margin: 20px 0 40px;
+}
+
+h3{
+	font-size:28px;
+	margin: 30px 0;
+}
+
+h4{
+	font-size:24px;
+	margin: 25px 0;
+}
+
+h5{
+	font-size:22px;
+	margin: 20px 0;
+}
+
+h6{
+	font-size:20px;
+	margin: 20px 0;
+}
+
+p{
+	margin: 0 0 40px;
+}
+
+blockquote{
+	font-family: 'sporting_grotesque', sans-serif;
+	font-size: 28px;
+    margin: 50px 0;
+    padding: 0 0 0 60px;
+    font-style: normal;
+}
+
+ul{
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+}
+li{
+    text-align: left;
+    position: relative;
+}
+li a{
+    display: inline-block;
+    text-decoration: none;
+    position: relative;
+}
+
+input {
+    border: 1px solid #00283C;
+    padding: 20px;
+    margin: 20px 0;
+    background: #fff;
+    display: block;
+    width: 300px;
+}
+input:focus{
+    border: 1px solid #bde513;
+    outline: none;
+}
+input::placeholder {
+    font-family: 'vremenagroteskbook-webfont', sans-serif;
+    color: #00283C;
+}
+input:focus::placeholder{
+    color:#bde513;
+}
+input[type="submit"] {
+    text-align: center;
+    padding: 20px 40px;
+    cursor: pointer;
+}
+input[type="submit"]:hover{
+    background-color: #00283C;
+    color: #fff;
+}
+input[type="submit"],
+input[type="submit"]:hover{
+    -webkit-transition: all 0.25s ease-in-out;
+    -moz-transition: all 0.25s ease-in-out;
+    -ms-transition: all 0.25s ease-in-out;
+    -o-transition: all 0.25s ease-in-out;
+    transition: all 0.25s ease-in-out; 
+}
+
+form#signin {
+    background-color: #fffff8;
+    border: 1px solid #bde513;
+    padding: 20px;
+}
+
+#signin input[type="text"],
+#signin input[type="email"]{
+    border:none;
+    border-bottom: 1px solid #bde513;
+    padding-left: 0;
+    background-color: transparent;
+}
+
+
+/* Links */
+
+a, a:visited, a:hover, a:focus, a:active {
```

Comment:
You [had put each selector on it's line](https://github.com/cafebabel/styleguide/pull/2/files#diff-ec123142b3b6b9db253da9d77a089e32R13) (which is great).
But here you write multiple selectors on one line.
Don't you want to do the same here?